roles/kubernetes/master/defaults/main.yml

```diff
@@ -137,3 +137,7 @@ kube_encryption_algorithm: "aescbc"
 
 # You may want to use ca.pem depending on your situation
 kube_front_proxy_ca: "front-proxy-ca.pem"
+
+# For the openstack integration kube-controller-manager needs to know
+# if cacert file exists in order to mount the file in the pod
+openstack_cacert: "{{ lookup('env','OS_CACERT') }}"
```

Comment:
Nice catch, but better put this variable to [kubespray-devfaults](https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubespray-defaults/defaults/main.yaml#L250-L265) and remove duplicate definition.

Reply:
then I also have to remove these lines https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes/node/defaults/main.yml#L114-L125 ?

Reply:
Not sure about all `openstak_*` variables, i understand kubespray defaults as set of variables which are used more than in one role. 
`openstack_cacert` good candidate for that place